src/routes/safe/components/Transactions/helpers/EditableTxParameters.tsx

```diff
@@ -73,7 +70,7 @@ export const EditableTxParameters = ({
       setSafeTxGas(txParameters.safeTxGas)
       setEthGasLimit(txParameters.ethGasLimit)
       setEthGasPrice(txParameters.ethGasPrice)
-      setEthMaxPrioFee(txParameters.ethMaxPrioFee || fromWei(DEFAULT_MAX_PRIO_FEE, 'gwei'))
+      setEthMaxPrioFee(txParameters.ethMaxPrioFee)
```

Comment:
Was the fallback really not needed?

Reply:
Now that it is being set relative to the estimation (and I intend to return the `DEFAULT_MAX_PRIO_FEE` in the `catch`), I don't think it's necessary. What do you think?

Reply:
As long as it works 🤷